content/agent/exporters/opencensus/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "OpenCensus"
+date: 2019-02-14T12:13:58-08:00
+logo: /images/opencensus-logo.png
+---
+
+- [Introduction](#introduction)
+- [Use cases](#use-cases)
+- [Configuration](#configuration)
+    - [Format](#format)
+    - [Example](#example)
+    - [Notes](#notes)
+- [References](#references)
+
+### Introduction
+
+ocagent's OpenCensus exporter allows it to export traces and metrics from ocagent to a backend of your choice
```

Comment:
nit: consider replacing "backend" with "service" since one Agent can export to another Agent, and generally we don't consider Agent as a backend.

Reply:
Gotcha, thanks!